.github/workflows/build.yml

```diff
@@ -27,8 +27,8 @@ jobs:
           #  experimental: false
           #  dependencies: 'highest'
 
-          # Experimental build for PHP 8
-          #- php-versions: '8.0'
+          # Experimental build for PHP nightly
+          #- php-versions: 'nightly'
```

Comment:
Just curious: Won't this fetch PHP8.1 soon?

Reply:
It's commented out right now :) But yes, this would potentially allow us to run builds against PHP-next early on. The reason it is outcommented is, that to make this viable it would need to not cause the build to fail akin to travis' "allow_failures" mode, which is not possible right now with GH actions.